### PR TITLE
Clarify UA Reduction OT enrollment for third-parties

### PR DESCRIPTION
**Pull request by abeyad**
Preview: https://deploy-preview-1950--developer-chrome-com.netlify.app/blog/user-agent-reduction-origin-trial/

### CODE REVIEW

File: site/en/blog/user-agent-reduction-origin-trial/index.md [hunk 1]
```diff
@@ -131,25 +134,30 @@ provided that the permissions policy allows it.
...
     demonstration of the origin trial (along with the source code).
 
-## How to participate in the origin trial as a third-party embed?
+## How to participate in the origin trial as a third-party embed? {: #enroll-third-party }
 
 Starting in Chrome 96, third-party embeds (for example, an iframe inside another site)
 can participate in the origin trial without requiring the top-level site to be
 enrolled.
 
 To enroll as a third-party embed, follow the same steps as for a [top-level site](#enroll-top-level),
```
Comment (Svenmay) [inline site/en/blog/user-agent-reduction-origin-trial/index.md:151]: Wouldn't it be better to remove this sentence? Following the steps from above (i.e. adding the OT token via header) does not make sense for third-party enrollment, no?
As described below, the token needs to be injected via JavaScript.

Comment (abeyad) [reply site/en/blog/user-agent-reduction-origin-trial/index.md]: Thanks Sven, I updated the text, please let me know if it reads more clear.

Comment (Svenmay) [reply site/en/blog/user-agent-reduction-origin-trial/index.md]: Yes, I do think this is better now. Thanks for fixing/rewriting this paragraph :-)

File: site/en/blog/user-agent-reduction-origin-trial/index.md [hunk 2]
```diff
@@ -131,23 +134,29 @@ provided that the permissions policy allows it.
         images or stylesheets) and subnavigations (for example, iframes) will send the
         reduced UA string, even if the origins of those requests are not
         enrolled in the origin trial.
-
+
```
Comment (heyawhite) [inline site/en/blog/user-agent-reduction-origin-trial/index.md:137]:
```suggestion

```

File: site/en/blog/user-agent-reduction-origin-trial/index.md [hunk 3]
```diff
@@ -131,23 +134,29 @@ provided that the permissions policy allows it.
...
 Starting in Chrome 96, third-party embeds (for example, an iframe inside another site)
 can participate in the origin trial without requiring the top-level site to be
 enrolled.
 
-To enroll as a third-party embed, follow the same steps as for a [top-level site](#enroll-top-level),
-except when [registering for an origin trial token](/origintrials/#/view_trial/-7123568710593282047),
-select the third-party token checkbox.  
+To enroll as a third-party embed:
+1.  Visit the 
+    [UserAgentReduction origin trial page](/origintrials/#/view_trial/-7123568710593282047)
```
Comment (heyawhite) [inline site/en/blog/user-agent-reduction-origin-trial/index.md:153]:
```suggestion
    [Trial for User Agent Reduction](/origintrials/#/view_trial/-7123568710593282047)
```

File: site/en/blog/user-agent-reduction-origin-trial/index.md [hunk 4]
```diff
@@ -131,23 +134,29 @@ provided that the permissions policy allows it.
...
+    and click on Register.
+1.  When creating the token, make sure to select the `Third-party matching` checkbox.
+1.  To receive the reduced User-Agent header from the third-party embed, follow the
```
Comment (heyawhite) [inline site/en/blog/user-agent-reduction-origin-trial/index.md:156]:
```suggestion
1.  To receive the reduced User-Agent header from the third-party embed,
```

File: site/en/blog/user-agent-reduction-origin-trial/index.md [hunk 5]
```diff
@@ -131,23 +134,29 @@ provided that the permissions policy allows it.
...
-To enroll as a third-party embed, follow the same steps as for a [top-level site](#enroll-top-level),
-except when [registering for an origin trial token](/origintrials/#/view_trial/-7123568710593282047),
-select the third-party token checkbox.  
+To enroll as a third-party embed:
+1.  Visit the 
+    [UserAgentReduction origin trial page](/origintrials/#/view_trial/-7123568710593282047)
+    and click on Register.
+1.  When creating the token, make sure to select the `Third-party matching` checkbox.
+1.  To receive the reduced User-Agent header from the third-party embed, follow the 
+    instructions for updating the HTTP response headers as outlined [above](#enroll-top-level).
```
Comment (heyawhite) [inline site/en/blog/user-agent-reduction-origin-trial/index.md:157]:
```suggestion
    [update the HTTP response headers](#enroll-top-level).
```

**Review by heyawhite — CHANGES_REQUESTED** [review]
Some language feedback.

**Review by heyawhite — APPROVED** [review]

**Comment by heyawhite** [comment]
Thanks @abeyad! I'll go ahead and merge.

**Comment by abeyad** [comment]
Thanks Alexandra!

**Pull request merged**